rusty_socket_server/src/handshake/response_line.rs

```diff
@@ -23,11 +23,11 @@ impl ResponseLine {
 
     pub fn build(request_key: &str) -> Self {
         let mut response_headers: HashMap<String, String> = HashMap::new();
-        let accpet_key = Self::generate_websocket_accept_key(request_key);
+        let accept_key = Self::generate_websocket_accept_key(request_key);
```

Comment:
I didn't know we can use Self to call methods without &self as method parameter. Nicely done.

Reply:
So, they are called associated function, you call associated function with namespace to self, but function that contains &self as it's parameters are methods.